node/core/provisioner/src/lib.rs

```diff
@@ -277,17 +284,9 @@ async fn send_inherent_data(
 	return_sender: oneshot::Sender<ProvisionerInherentData>,
 	mut from_job: mpsc::Sender<FromJob>,
 ) -> Result<(), Error> {
-	let availability_cores = match request_availability_cores(relay_parent, &mut from_job)
-		.await?
+	let availability_cores = request_availability_cores(relay_parent, &mut from_job)
 		.await?
-	{
-		Ok(cores) => cores,
-		Err(runtime_err) => {
-			// Don't take down the node on runtime API errors.
-			log::warn!(target: "provisioner", "Encountered a runtime API error: {:?}", runtime_err);
-			return Ok(());
-		}
-	};
```

Comment:
@coriolinus it seems to we early return on other errors and don't take down the node, but print a warning in this case already, so I removed this code. Let me know if I was wrong.